custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -60,6 +59,16 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun addAttachment() {
```

Comment:
This doesn't return the type expected by the interface.
I highly recommend compiling and running the test locally.

Reply:
removed